### DISCUSSION TITLE
Why does @rollup/plugin-commonjs look at /root/.node_modules?

### DISCUSSION THREAD

**Discussion by hgezim** [rollup/plugins#1250] [Q&A]
I'm running it inside docker and getting:

```
[!] (plugin commonjs--resolver) Error: EACCES: permission denied, stat '/root/.node_modules/@org/package/dist/esm'
embedded    | Error: EACCES: permission denied, stat '/root/.node_modules/@org/package/dist/esm'
```

Any ideas how to get around this?

**Comment by bytestream** [comment]
I haven't specifically looked at the `commonjs--resolver` plugin but I had the same issue with the `node-resolve` plugin:
```
[!] (plugin node-resolve) Error: EACCES: permission denied, stat '/root/.node_modules/...'
```

It looks at `/$HOME/.node_modules` as a result of https://github.com/browserify/resolve/pull/273, presumably following https://nodejs.org/api/modules.html#loading-from-the-global-folders

It can be resolved in the `node-resolve` plugin by specifying `modulePaths: []`. This is supposed to be the default per the README, but that's not true and I opened https://github.com/rollup/plugins/pull/1534 to correct that.
